tutorials/fluidSolidInteraction/HronTurekFsi3/force.gnuplot

```diff
@@ -9,5 +9,5 @@ set grid
 set y2tics
 
 plot [0.01:] \
-    "< sed s/[\\(\\)]//g ./postProcessing/fluid/forces/0/force.dat" using 1:($2+$5)/0.015 axis x1y1 title "Fx" with lines, \
-    "< sed s/[\\(\\)]//g ./postProcessing/fluid/forces/0/force.dat" using 1:($3+$6)/0.015 axis x1y2 title "Fy" with lines
\ No newline at end of file
+    "< sed s/[\\(\\)]//g `find . -name 'forces.dat'`" using 1:($2+$5)/0.015 axis x1y1 title "Fx" with lines, \
```

Comment:
This seems fine if it works.

Ideally, we would avoid such complicated expressions, but I agree with you that this may be the best solution in this case.

Reply:
I agree, but lets say in case of having a FSI case where you are getting the force on the fluid region for example, we can't have the same path as we have In non-FSI cases. there are 3 cases right now which they have force post processing, _HronTurek_, _beamInCrossFlow_ and _cylinderInChannel_ which they don't have a same path for `force.dat`.
I thought a lot on how we can avoid this `find`, but this one actually seems to be an efficient one. Another alternative would be:
1.  look if there is any force postprocessing
2.  `if ==TRUE`, look for **`OF_VERSION`** and **`functionObjectsDict`**
3.  generate a `forces.gnuplot` based on previous step(Assign the path and plot configs.)

but I have no idea how complicated(developer-end and user-end) it would be.

Reply:
Sure, I understand. It is a situation where there are many permutations.

From the user perspective, it is probably better that we keep this find step directly in the gnuplot script as you have done; then if someone is interested in gnuplot they can understand what is going on.

Let's stick with this!